File: java/URI_1071.java
package URI;
import java.util.Scanner;

public class URI_1071 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int x = sc.nextInt();
		int y = sc.nextInt();
		int sum = 0;
		
		for(int i = x - 1; i >= y + 1; i--) {
			if(i % 2 != 0) {
				sum += i;
			}			
		}
		System.out.println(sum);
		
		
		sc.close();

	}

}

File: kotlin/URI_1071.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val x = sc.nextInt()
    val y = sc.nextInt()
    var soma = 0

    val range = if(x > y) (y+1) until x else (x+1) until y
    for (i in range) {
        if (i % 2 != 0) soma += i
    }

    println(soma)
}
